### DISCUSSION TITLE
infrastructure/proxmox/gpu-passthrough/gpu-passthrough-to-vm/

### DISCUSSION THREAD

**Discussion by giscus** [fire1ce/3os.org#98] [General]
# infrastructure/proxmox/gpu-passthrough/gpu-passthrough-to-vm/

Proxmox full gpu passthrough to VM configuration for hardware acceleration.

https://3os.org/infrastructure/proxmox/gpu-passthrough/gpu-passthrough-to-vm/

**Comment by Softlution2** [comment]
Here is a guide troubleshooting xorg error in proxmox installation https://youtu.be/-6fRTpmmuHs
